packages/rps/.env.production

```diff
@@ -1,8 +1,14 @@
-TARGET_NETWORK='ropsten'
 CHAIN_NETWORK_ID=3
-FIREBASE_PROJECT='rock-paper-scissors-production' 
-FIREBASE_API_KEY='AIzaSyCGck4vYsR1CF6LEiOH2HtYnUNYgs_tYbk' 
-WALLET_URL='https://wallet.statechannels.org' 
+
+TARGET_NETWORK=ropsten
+
+HUB_ADDRESS=0x2c2ED965A624156Ab3cc60F3cbe63B9d196bF036
```

Comment:
does `rps` need to know this?

Reply:
Uhmmmmm, no, the wallet does, let me fix that.